webapp/src/Controller/API/GeneralInfoController.php

```diff
@@ -91,20 +94,33 @@ public function getVersionAction(): array
      *     description="Information about the API and DOMjudge",
      *     @OA\JsonContent(
      *         type="object",
-     *         @OA\Property(property="api_version", type="integer"),
-     *         @OA\Property(property="domjudge_version", type="string"),
-     *         @OA\Property(property="environment", type="string"),
-     *         @OA\Property(property="doc_url", type="string")
+     *         @OA\Property(property="version", type="string", description="Version of the CCS Specs Contest API the API adheres to"),
+     *         @OA\Property(property="version_url", type="string", description="URL with the speification of the Contest API"),
```

Comment:
Why did we not catch a typo here?
```suggestion
     *         @OA\Property(property="version_url", type="string", description="URL with the specification of the Contest API"),
```